.github/workflows/run_tests.yml

```
@@ -69,8 +69,7 @@ jobs:
           python-version: "3.10"
 
       - name: Install the project
-        run: uv sync --extra test --resolution lowest
+        run: uv sync --resolution lowest-direct
```

```suggestion
        run: uv sync --extra dev --resolution lowest-direct
```

Just looked at the uv docs. It picks up the dev dependency group automatically, but not the optional dependency. Maybe this will fix it?